website/homepage/src/docs/obj_doc_template.html

```diff
@@ -128,11 +128,11 @@ <h4 class="mt-4 p-3 font-semibold">Example Usage</h4>
     <h4 class="mt-4 p-3 font-semibold">Methods</h4>
       <div class="flex flex-col gap-8 pl-12">
         {% for fn in obj["fns"] %}
-          {% with obj=fn, is_class=False, parent=parent + "." + obj["name"] %}
+          {% with obj=fn, is_component=False, parent=parent + "." + obj["name"] %}
             {% include "docs/obj_doc_template.html" %}
           {% endwith %}
         {% endfor %}
-      <div class="ml-12">
+      <div class="ml-12"> </div>
```

Comment:
Pre-existing bug. Need to close the div here otherwise the components are all nested within the `textbox` div and the current-nav-link logic gets thrown off.